src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/fine/DebtPositionFineClient.java

```diff
@@ -5,5 +5,5 @@
 
 public interface DebtPositionFineClient {
 
-  String handleFineReductionExpiration(Long debtPositionId, PaymentEventRequestDTO paymentEventRequestDTO, FineWfExecutionConfig executionParams);
+  String handleFineReductionExpiration(Long debtPositionId, PaymentEventRequestDTO paymentEventRequestDTO, boolean massive, FineWfExecutionConfig executionParams, String accessToken);
```

Comment:
I expect it should need just these parameters!
```suggestion
  String expireFineReduction(Long debtPositionId, FineWfExecutionConfig executionParams);
```